.gitignore

```diff
@@ -34,3 +34,6 @@ sysinfo.txt
 # Builds
 *.apk
 *.unitypackage
+.idea/
+Assets/ScriptableObjects/Grid/TestGridData.asset.meta
```

Comment:
These two files should not be in .gitignore.
Normally you put files that you will be creating (and only you) in your global gitignore list (right click in sourcetree file and click "ignore...")

Reply:
That's exactly what I did, but I will check to make sure that those changes go to my global .gitignore instead of the one in the project.